addon/components/listing/table.hbs

```diff
@@ -0,0 +1,65 @@
+<div ...attributes>
+  <AuTable class="listing-table">
+    {{! TODO: Named blocks can't be rendered conditionally yet, so we would have to duplicate the component and all the other blocks as a workaround.
+        Since the title is only a "nice to have" we'll disable it for now until a better solution is found.
+    }}
+    {{!--
+    <:title>
+      {{this.title}}
+    </:title>
+    --}}
```

Comment:
I've disabled the table "title" for now since it looks weird if it isn't set and we would need to duplicate most of the template if we want to render it conditionally.